tests/test_data/adaptation/input/losses/traffic_intensities.csv

```diff
@@ -0,0 +1,273 @@
+ID,evening_peak_total,evening_peak_freight,evening_peak_commute,evening_peak_business,evening_peak_other,day_freight,day_commute,day_business,day_other,day_total
```

Comment:
Question: where does data come from? 
Is it public data that you found in the repo?

Reply:
This data was provided by @Cham8920 . I guess it's fake data, isn't it?

Reply:
Yes its fake data for Beira.